packages/storybook/src/nimble/tabs/tabs.stories.ts

```diff
@@ -42,18 +44,58 @@ const metadata: Meta<TabsArgs> = {
     }
 };
 
+const simpleTabs: readonly TabArgs[] = [
+    { title: 'Tab 1', id: '0', disabled: false },
```

Comment:
These ids are 0-indexed while the activeid choices are 1-indexed, can we make all these consistent one way or another, i.e. only show and use indexes or positions for id / label text?

Reply:
Re-opening because this does not seem addressed consistently, see the following:
![image](https://github.com/user-attachments/assets/53bcf765-c1b5-4548-a804-55645c8d2381)